pkg/config/config.go

```diff
@@ -34,19 +35,50 @@ type IgnoreEntry struct {
 	Reason      string    `toml:"reason"`
 }
 
+type IgnorePackageVersionEntry struct {
+	Name      string `json:"name"`
+	Version   string `json:"version"` // Q: If empty all versions of the package are ignored?
+	Ecosystem string `json:"ecosystem"`
+	// TODO: Commit? Should we use models.PackageInfo? Or keep this config
+	// decoupled from that model. I lean towards keeping it decoupled.
+	IgnoreUntil time.Time `toml:"ignoreUntil"`
+	Reason      string    `toml:"reason"`
+}
+
 func (c *Config) ShouldIgnore(vulnID string) (bool, IgnoreEntry) {
 	index := slices.IndexFunc(c.IgnoredVulns, func(elem IgnoreEntry) bool { return elem.ID == vulnID })
 	if index == -1 {
 		return false, IgnoreEntry{}
 	}
 	ignoredLine := c.IgnoredVulns[index]
-	if ignoredLine.IgnoreUntil.IsZero() {
+
+	return shouldIgnoreTimestamp(ignoredLine.IgnoreUntil), ignoredLine
+}
+
+func (c *Config) ShouldIgnorePackageVersion(name, version, ecosystem string) (bool, IgnorePackageVersionEntry) {
+	index := slices.IndexFunc(c.IgnoredPackageVersions, func(elem IgnorePackageVersionEntry) bool {
+		if name != elem.Name {
+			return false
+		}
+
+		return version == elem.Version || elem.Version == ""
```

Comment:
> elem.Version == ""

I think this is helpful behaviour, let me know if you think this would be unexpected for any reason.

Reply:
This makes sense to me! If no version is defined, all versions are affected?

Reply:
yes that's right.